src/util/getOracleCredentials.ts

```diff
@@ -0,0 +1,21 @@
+import { SecretsManager } from 'aws-sdk';
+import { DatabaseConnectionConfig } from '../interfaces/DatabaseConnectionConfig';
+import localOracleConfig from '../config/localOracleConfig.json';
+
+import logger from './logger';
+
+export const getOracleCredentials = async (
+  secretName: string,
+): Promise<DatabaseConnectionConfig> => {
+  if (!process.env.ORACLE_CONFIG_SECRET) return localOracleConfig;
```

Comment:
Can we remove localOracleConfig in favour of returning a DatabaseConnectionConfig populated from the environmental variables in the env file? localOracleConfig and DatabaseConnectionConfig are identical objects and it will keep configuration in the env file which is omitted from source control.